.github/workflows/assign-reviewer.yml

```diff
@@ -0,0 +1,27 @@
+name: Assign Random Reviewer
```

Comment:
we could consolidate all this into one .yml in future. For now this looks good. Thanks for removing the cache files. 
I have not being able to test this as it still show suggested on my last PR. Since this won't break any app functionality I will approve it and observe whether it works on my next PR.